open3d_conversions/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>open3d_conversions</name>
+  <version>0.0.0</version>
+  <description>Provides conversion functions to and from Open3D datatypes</description>
+
+  <maintainer email="matnay17@gmail.com">Pranay Mathur</maintainer>
```

Comment:
Add me too (since I sort of volunteered myself into at least helping get this all off the ground)

Reply:
Yep, already added on line 9 :slightly_smiling_face: